ci-operator/step-registry/ignition/bastionhost/ignition-bastionhost-commands.sh

```diff
@@ -85,6 +85,14 @@ EOF
 cat > ${workdir}/squid.conf << EOF
 auth_param basic program /usr/lib64/squid/basic_ncsa_auth /etc/squid/passwords
 auth_param basic realm proxy
+#prfsr90
+delay_pools 1
+delay_class 1 1
+#delay_parameters 1 10000000/10000000 -1/-1 -1/-1
+delay_parameters 1  8750000/8750000 8750000/8750000 8750000/8750000
+delay_access 1 allow all
+http_access allow all
```

Comment:
@SachinNinganure  DO NOT all all aceess without authentication

Reply:
changes done!! there is no proxy to terminate